homeassistant/helpers/device_registry.py

```diff
@@ -263,6 +263,9 @@ def disabled(self) -> bool:
     @property
     def dict_repr(self) -> dict[str, Any]:
         """Return a dict representation of the entry."""
+        # Convert sets and tuples to lists
```

Comment:
This one had them all done already. I added the comment so we don't forget if we add more in the future